lte/gateway/c/core/oai/tasks/amf/amf_app_state_manager.cpp

```diff
@@ -159,12 +152,65 @@ void AmfNasStateManager::amf_nas_state_init_local_state() {
  */
 amf_app_desc_t* AmfNasStateManager::get_state(bool read_from_redis) {
   state_dirty = true;
-
+#if MME_UNIT_TEST
```

Comment:
should not this be `!MME_UNIT_TEST`?

Reply:
also again looks like you handle this build flag inside the call `read_state_from_db()`. it should not be here then.

Reply:
Addressed this review comment.